src/stactools/noaa_sst/stac.py

```diff
@@ -30,77 +41,78 @@ def create_collection() -> Collection:
     Returns:
         Collection: STAC Collection object
     """
-    providers = [
-        Provider(
-            name="The OS Community",
-            roles=[
-                ProviderRole.PRODUCER, ProviderRole.PROCESSOR,
-                ProviderRole.HOST
-            ],
-            url="https://github.com/stac-utils/stactools",
-        )
-    ]
-
-    # Time must be in UTC
-    demo_time = datetime.now(tz=timezone.utc)
-
     extent = Extent(
-        SpatialExtent([[-180., 90., 180., -90.]]),
-        TemporalExtent([demo_time, None]),
+        SpatialExtent([SPATIAL_EXTENT]),
+        TemporalExtent(TEMPORAL_EXTENT),
     )
 
     collection = Collection(
-        id="my-collection-id",
-        title="A dummy STAC Collection",
-        description="Used for demonstration purposes",
-        license="CC-0",
-        providers=providers,
+        id=NOAA_SST_ID,
+        title=TITLE,
+        description=DESCRIPTION,
+        license=LICENSE,
+        providers=[SST_PROVIDER],
         extent=extent,
         catalog_type=CatalogType.RELATIVE_PUBLISHED,
     )
 
```

Comment:
Consider adding the item assets extension to collection: https://github.com/stac-extensions/item-assets

Reply:
Added